### DISCUSSION TITLE
Whenever someone raises a PR, it requires me to authorize it only then it gets deployed

### DISCUSSION THREAD

**Discussion by Mayank0255** [vercel/community#718] [Help] [answered]
What I want is to give the permission to deploy whenever someone raises a PR, and not require me to authorize it each time

**Comment by amyegan** [comment, marked as answer]
Hi @Mayank0255. Thanks for reaching out! This feature is called [Git Fork Protection](https://vercel.com/docs/concepts/projects/overview#git-fork-protection) and it is enabled by default to keep your project safe from leaking sensitive project info. It is possible to disable the feature from the Dashboard in Project -> Settings -> Security where you can toggle Git Fork Protection to "disabled".

**Reply by dj-stormtrooper** [reply]
Hi @amyegan! I found in the documentation that Git Fork Protection should be skipped when one of the team members raise a deployment from a fork
https://vercel.com/docs/concepts/git#deploying-forks-of-public-git-repositories

But we can't get it to work that way, it always forces me to approve deployment for my own pull request (as a team member).

Can you confirm that this is the way it should work? Maybe there is a workaround or additional settings? 

Thank you!

**Reply by amyegan** [reply]
Right, anyone who is recognized as part of the Team which owns the Project should not have deployments blocked by Git Fork Protection. Sorry to hear this is still a problem for you. Is it happening for everyone on your Team, or only for some peoplel?

Email metadata mismatch is the most common reason I have seen for GitHub commits not being recognized as coming from someone on the team. One quick thing should check is that you have [set your email metadata to a matching email address](https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata) so the commits can be verified as yours.

**Reply by dj-stormtrooper** [reply]
Thanks for the answer! I think it affects whole team (both owners and members), not only a few people

What I've tried:
* deploy non-protected project (without environment variables) from this fork - ✅ and it shows my github-account as an author
* change email in local config, commit and create PR - ❌
* commit directly from Github UI, from the same account that was used for Vercel login - ❌

Still can't make a deploy without auth approval. My email in members list is the same as in github

Example PR
https://github.com/cloudquery/cloudquery/pull/1470

**Reply by amyegan** [reply]
This certainly seems like unintended behavior. We're going to need to dig into more account details to solve this, and a private communication channel seems more appropriate. Could you create a support ticket on behalf of your team at https://vercel.com/help?

**Reply by dj-stormtrooper** [reply]
Sure, thanks for your help!
